docs/ui/prepare-migration/dataflow-config.md

```diff
@@ -0,0 +1,47 @@
+---
+layout: default
+title: Configure Dataflow (Optional)
+parent: Prepare Migration Page
+grand_parent: SMT UI
+nav_order: 4
+---
+
+# Setting up connection profiles
+{: .no_toc }
+
+In case of minimal downtime migration, the dataflow jobs launched by the Spanner Migration Tool can be OPTIONALLY configured with custom runtime environment variables such as MaxWorkers, NumWorkers etc. 
+
+{: .highlight }
+The step shows configured (via a green tick) by default, denoting it is optional to configure.
+
+To configure dataflow, first specify the target database in the 'Configure Spanner Database' step. This enables the configure button for the remaining steps. The configurable fields for Dataflow are mentioned below.
```

Comment:
We should add the link of the Dataflow documentation which talks about networks and subnetworks.

Reply:
We should also add reference to general Dataflow configuration parameters (which may not be part of the form) here for completeness.